packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -120,6 +132,29 @@ export class ModelInstanceDocument<T = Record<string, any>> extends Stream {
     )
   }
 
+  /**
+   * Creates a deterministic Tile document.
+   * @param ceramic - Instance of CeramicAPI used to communicate with the Ceramic network
+   * @param metadata - Genesis metadata
+   * @param opts - Additional options
+   */
+  static async deterministic<T>(
```

Comment:
I also considered calling this method `single()`, curious what you think about that @PaulLeCam

Reply:
I like `single()`, should be more explicit considering that's the term we use for the relation, but I don't feel strongly about it so no problem keeping as `deterministic()`.

Reply:
@oed any opinion on `single()` vs `deterministic()`.  I think I very slightly lean towards `single`?  But it's very slight.

Reply:
`single` seems nice